build.xml

```diff
@@ -98,8 +97,8 @@
     <target name="standards-fix-packages" description="Automatically fixes some coding standards violations in all packages.">
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
```

Comment:
The flag `--verbose` is not used here, but it is used in other build definitions. Can you explain or fix it?

Reply:
verbose was added